cmd/contact-exporter/main.go

```diff
@@ -52,6 +53,33 @@ func (c contactExporter) findContacts() ([]contact, error) {
 	return contactsList, nil
 }
 
+func (c contactExporter) findContactsForDomains(domains []string) ([]contact, error) {
+	var contactsList []contact
+	for _, domain := range domains {
+		// Pass the same list in each time, gorp will happily just append to the slice
+		// instead of overwriting it each time
+		// https://github.com/coopernurse/gorp/blob/9cd2b5ef5b82fde4e7c51776ac3f94398b8af076/gorp.go#L1644-L1651
+		_, err := c.dbMap.Select(
+			&contactsList,
+			`SELECT registrationID as id FROM certificates
+                         WHERE expires >= :expireCutoff AND
+                         serial IN (
+                           SELECT serial FROM issuedNames
+                            WHERE reversedName = :reversedName
+                         )`,
+			map[string]interface{}{
+				"expireCutoff": c.clk.Now().Add(-c.grace),
+				"reversedName": sa.ReverseName(domain),
+			},
+		)
+		if err != nil {
```

Comment:
I think you would want to check for `err != sql.ErrNoRows` here to make this a little more user-friendly if someone happens to enter in some domains that don't have any contacts associated with them. Am I wrong?